templates/3d_compute_median_pulse_height.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Move defaults to function defs, not at top of cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>